tests/testthat/test-plotreg.R

```diff
@@ -0,0 +1,53 @@
+# testthat for plotreg
+
+# linear model -pvalues
+ctl <- c(4.17,5.58,5.18,6.11,4.50,4.61,5.17,4.53,5.33,5.14)
+trt <- c(4.81,4.17,4.41,3.59,5.87,3.83,6.03,4.89,4.32,4.69)
+group <- gl(2, 10, 20, labels = c("Ctl", "Trt"))
+weight <- c(ctl, trt)
+m1 <- lm(weight ~ group)
+m2 <- lm(weight ~ group - 1)
+p <- plotreg(list(m1,m2))
+p2 <- plotreg(list(m1,m2), use.se = TRUE)
+# tests if dataframe is correctly made
+test_that("Dataframe is correctly constructed",{
+    expect_true(is.ggplot(p))
+    expect_equal(p$data, readRDS("../files/PlotDataFrame.RDS"))
+})
+
+# tests if it is printing the correct xlab
```

Comment:
Can you move these comments into the first argument of the `test_that` call (in all instances)? In this case, for example, this would look as follows: `test_that("plotreg prints the correct xlab'", {`. Delete the comment once moved into the `test_that` command. (Also note the additional spaces before `{`.)